tests/unit/reports/single_table/_properties/test_boundary.py

```diff
@@ -0,0 +1,101 @@
+from unittest.mock import Mock, call, patch
+
+import numpy as np
+import pandas as pd
+
+from sdmetrics.reports.single_table._properties.boundary import Boundary
+
+
+class TestBoundary:
+
+    @patch('sdmetrics.reports.single_table._properties.boundary.BoundaryAdherence.compute')
+    def test__generate_details(self, boundary_adherence_mock):
```

Comment:
can we add one unit test where the metric is mocked to throw an error so we can test the exception handling?

Reply:
Yes, done in [e260202](https://github.com/sdv-dev/SDMetrics/pull/400/commits/e260202e27f1610cbe8b540aa6b054d0e72f4851)